tfx/components/trainer/executor.py

```diff
@@ -302,12 +313,14 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
           export_dir_base=fn_args.eval_model_dir,
           eval_input_receiver_fn=training_spec['eval_input_receiver_fn'])
 
-      # TODO(b/158106209): refactor serving_model_dir to only contain model.
-      # Copy model run information to ModelRun artifact
-      io_utils.copy_dir(fn_args.serving_model_dir, fn_args.model_run_dir)
-
       absl.logging.info('Exported eval_savedmodel to %s.',
                         fn_args.eval_model_dir)
+
+      # TODO(b/160795287): Deprecate estimator based executor.
+      # Copy serving and eval model from model_run to model artifact directory.
+      dir_utils.copy_model(fn_args.model_run_dir, serving_dest, 'serving')
+      dir_utils.copy_model(fn_args.model_run_dir, eval_dest, 'eval')
+
     else:
       absl.logging.info(
           'eval_savedmodel export for TFMA is skipped because '
```

Comment:
update this 

change to model export is skipped ....

Reply:
I mean just leave one log info

"model export is skipped because this is not the chief worker"